docs/index.md

```diff
@@ -7,22 +7,23 @@ hide:
 
 Here we are documenting the processes and work of the [AI Validation Team](about/team.md) at the [Ministry of the
 Interior and Kingdom Relations](https://www.government.nl/ministries/ministry-of-the-interior-and-kingdom-relations) in
-The Netherlands.
+The Netherlands. 
 
-We are a team of mostly engineers at a policy department.
+We are a team of engineers, UX design and researchers, and product people at a policy department.
 
-We work on the projects within the _Transparency of Algorithmic Decision making_ scope:
+We work on the following projects within our scope:
 
 ``` mermaid
 graph TB
     ak[<a href='https://minbzk.github.io/Algoritmekader/'>Algoritmekader</a>] --> tad
 
-    subgraph Transparency of Algorithmic Decision making
+    subgraph AI Validation Team
```

Comment:
The Algorithm Management Toolkit will contain a set of tools, a.o. the platform, reporting format, LLM Benchmark Tooling, CI/CD tools, etc. That's why I propose to label this box AMT.

Another reason: the AI Validation team also works on the Algoritmekader and publishing information in registers.

```suggestion
    subgraph Algorithm Management Toolkit
```